options/options.cpp

```diff
@@ -587,6 +591,37 @@ const option::Descriptor usage[] = {
     Arg::Numeric,
     "  --kind-bound-step \tAmount by which bound (unrolling depth) is "
     "increased in k-induction (default: 1)" },
+  { NO_INTERP_FRONTIER_SIMPL,
+    0,
+    "",
+    "no-interp-frontier-simpl",
+    Arg::None,
+    "  --no-interp-frontier-simpl \tDisable frontier-set simplification in "
+    "interp engine" },
+  { INTERP_PROPS,
+    0,
+    "",
+    "interp-props",
+    Arg::NonEmpty,
+    "  --interp-props \tSpecifies at which time frames properties are "
+    "considered when computing interpolants: all, first-and-last, only-last "
+    "(WARNING: choosing not 'all' could cause incompleteness on some "
+    "instances; the fixed point derived by 'only-last' might contain "
+    "unreachable bad states)" },
+  { INTERP_EAGER_UNROLL,
+    0,
+    "",
+    "interp-eager-unroll",
+    Arg::None,
+    "  --interp-eager-unroll \tUnroll the transition system eagerly in interp "
+    "engine" },
+  { INTERP_BACKWARD,
+    0,
+    "",
+    "interp-backward",
+    Arg::None,
+    "  --interp-backward \tCompute interpolants in a backward manner in interp "
```

Comment:
please add text about the other case when this option is not use. something like " by default interpolates are computed ..... "